src/server/req_delete.c

```diff
@@ -394,6 +394,7 @@ req_deletejob(struct batch_request *preq)
 				parent->ji_ajtrk->tkm_tbl[offset].trk_substate =
```

Comment:
We are updating the deleted jobs count when find_job() returns NULL. What should happen in the case when find_job returns a valid pointer. Do we need to adjust the count for that case as well.